src/functional/ffront/foast_passes/type_deduction.py

```diff
@@ -78,15 +78,15 @@ class FieldOperatorTypeDeduction(NodeTranslator):
     ---------
     >>> import ast
     >>> from functional.common import Field
-    >>> from functional.ffront.source_utils import SourceDefinition, ClosureRefs
+    >>> from functional.ffront.source_utils import SourceDefinition, CapturedVars
     >>> from functional.ffront.func_to_foast import FieldOperatorParser
     >>> def example(a: "Field[..., float]", b: "Field[..., float]"):
     ...     return a + b
 
     >>> sdef = SourceDefinition.from_function(example)
-    >>> cref = ClosureRefs.from_function(example)
+    >>> cref = CapturedVars.from_function(example)
```

Comment:
Make sure to handle a potential conflict with the `cref` name, since it has been also renamed in other PR.
```suggestion
    >>> c_vars = CapturedVars.from_function(example)
```